libImaging/Resample.c

```diff
@@ -184,7 +184,6 @@ precompute_coeffs(int inSize, int outSize, struct filter *filterp,
                 } else if (x == xmax - 1) {
                     xmax -= 1;
                 }
-                continue;
             }
             k[x] = w;
```

Comment:
Yeah, I'm not seeing how that happens. xmax should be decremented on any 0 coefficient, and then the last loop should go from that decremented xmax to kmax. 

It does look like a calloc instead of malloc would solve all the initialization problems in one less math prone statement:
```
    kk = calloc(outSize * kmax , sizeof(double));
```